api/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -44,6 +44,8 @@ type DatadogAgentSpec struct {
 type DatadogFeatures struct {
 	// Application-level features
 
+	// OTelAgent configuration.
+	OTelAgent *OTelAgentFeatureConfig `json:"otelAgent,omitempty"`
```

Comment:
is `otelAgent` really best name for this feature? should this be `otelCollector` or just `otel`? other features don't mention 'Agent'.

Reply:
Renamed feature from `otelAgent` to `otelCollector`.